advanced/accessors/01_accessor_examples.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<blockquote>attaching a custom function to xarray types</blockquote><p>I think to be clearer, "xarray types" should change to "xarray data structure (<code>DataArray</code> or <code>Dataset</code>)" since there's only the two type of accessors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>